apps/music/views/album-detail/index.html

```diff
@@ -29,9 +29,9 @@
 <body role="application">
   <gaia-fast-list id="list">
     <template>
-      <a href="/player?id=${name}" data-file-path="${name}">
+      <a href="${url}" data-file-path="${name}">
```

Comment:
I *think* we can hard-code simply `href="/player"` here and we don't need the `url` property in this case.